eclass/acct-user.eclass

```diff
@@ -231,6 +231,7 @@ acct-user_pkg_postinst() {
 	# NB: eset* functions check current value
 	esethome "${ACCT_USER_NAME}" "${ACCT_USER_HOME}"
 	esetshell "${ACCT_USER_NAME}" "${ACCT_USER_SHELL}"
+	eunlockuser "${ACCT_USER_NAME}"
```

Comment:
We don't have a way to distinguish between "locked by the administrator" and "locked because the package was depcleaned last week", so as a courtesy I think we should `ewarn` here if we unlock an existing account.

Reply:
I was thinking of unlocking it only when installing fresh but then it'd be silly if rebuild didn't restore expected status.

Reply:
Sure, but for twenty years, locking has been a reliable way to "turn off" an account in Gentoo. We're going to change that, so we should make some noise about it.

Reply:
I'm still wondering if there's a 'better way'. If we're going to be noisy, it would probably make sense to be noisy if the account was locked indeed. I'll see if I can sensibly update the function to distinguish that.